pokemongo_bot/cell_workers/sniper.py

```diff
@@ -331,6 +332,10 @@ def snipe(self, pokemon):
                 if (seconds_since_last_check < self.MIN_SECONDS_ALLOWED_FOR_CELL_CHECK):
                     time.sleep(self.MIN_SECONDS_ALLOWED_FOR_CELL_CHECK - seconds_since_last_check)
 
+                # Sleep a bit longer for the Pokemon to appear
+                self._log('Waiting for the Pokemon to appear...')
+                time.sleep(randint(5,10))
```

Comment:
I understand that we have to there is a limit of 10 secs for cell check. What I am referring to is the duration of sleep after teleleporting. Right now, it is 3 seconds in _teleport function and that is too fast for the Pokemon to appear. With current setting at 3 sec, Bot misses almost half of the Pokemon. Once I increased the sleep time after teleporting to the destination, bot caught 100% of all Pokemons. I have a confirmed source of all 100IV Pokemon and I've tested with several accounts to confirm. Let me know if you want to use my Pokemon list (URL) for testing. 

We don't want to increase the sleeping time here because it applies to both teleport to the Pokemon and back. We only want to increase the time after teleporting to the Pokemon.

def _teleport(self, latitude, longitude, altitude):
        self.bot.api.set_position(latitude, longitude, altitude, True)
        time.sleep(3)